app/views/propertyLinking/uploadRatesBillLeaseOrLicense.scala.html

```diff
@@ -109,7 +109,7 @@ <h1 id="page-header" class="govuk-heading-l govuk-!-margin-bottom-3">@title</h1>
                     @upload.fileMetadata.toDisplayFileName
                 </dd>
                 <dd class="govuk-summary-list__actions">
-                    <button id="remove-file" type="submit" class="govuk-body govuk-link" formaction="@UploadController.remove(fileReference, evidenceChoice)">
+                    <button id="remove-file" type="submit" class="govuk-body govuk-git adlink govuk-!-static-margin-bottom-0" formaction="@UploadController.remove(fileReference, evidenceChoice)">
```

Comment:
seems to me `git ad` was typed here by accident

Reply:
you are correct, thank you. I've pushed a new commit